packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -4,6 +4,7 @@ import classNames from 'classnames/bind';
 import ButtonGroup from 'terra-button-group';
 import Menu from './_CollapsibleMenu';
 import styles from './CollapsibleMenuView.module.scss';
+import CollapsibleMenuView from '../lib/CollapsibleMenuView';
```

Comment:
Why are we going importing from the lib folder? These are in the same folder so we can just do:

```suggestion
import CollapsibleMenuView from './CollapsibleMenuView';
```